assets/xml/objects/object_jya_iron.xml

```diff
@@ -0,0 +1,13 @@
+<Root>
+    <File Name="object_jya_iron" Segment="6">
+        <DList Name="gPillarDL" Offset="0x0240"/>
+        <DList Name="gObjectJyaIronDL_000600" Offset="0x0600"/>
+        <DList Name="gObjectJyaIronDL_000880" Offset="0x0880"/>
+        <DList Name="gObjectJyaIronDL_000AE0" Offset="0x0AE0"/>
+        <Collision Name="gPillarCol" Offset="0x0D48"/>
+        <DList Name="gThroneDL" Offset="0x1050"/>
```

Comment:
Looks good! However I'd suggest maybe specifying IronKnuckle in these, just since there could potentially be another "pillar" in the game that might conflict in some circumstances.

Reply:
I don't even have that file in this pr :confused:

Reply:
Oh weird, it seems to show up in the diff, still worth changing though :V